task8/TutterApp/src/main/java/dao/HashTagDAOImpl.java

```diff
@@ -0,0 +1,107 @@
+package dao;
```

Comment:
Technically, most of the time, `...impl` classes are placed in a subpackage named, also, `impl`, and not side-by-side with the interfaces. A little tradition..